src/libpgmoneta/json.c

```diff
@@ -613,6 +615,96 @@ pgmoneta_json_get_value(struct json* item, char* key)
    return NULL;
 }
 
+int8_t
+pgmoneta_json_get_int8_value(struct json* item, char* key)
+{
+   if (item == NULL)
+   {
+      return 0;
+   }
+   struct json_value* val = pgmoneta_json_get_value(item, key);
```

Comment:
Can you move `struct json_value* val = NULL;` to the beginning of the function ?